posthog-android/src/main/java/com/posthog/android/replay/PostHogReplayIntegration.kt

```diff
@@ -312,6 +336,9 @@ public class PostHogReplayIntegration(
 
         // workaround for react native that is started after the window is added
         // Curtains.rootViews should be empty for normal apps yet
+
+        sessionStartTime = config.dateProvider.currentTimeMillis()
```

Comment:
Here is just the setup, the session actually only starts when the first snapshot is captured, and that can be a console log, a network log, or a screenshot/wireframe snapshot.
Search for the usage of this extension https://github.com/PostHog/posthog-android/blob/351a86b7c7caf9c72aad684d3721fad2872cfa2c/posthog/src/main/java/com/posthog/internal/replay/RRUtils.kt#L7